modules/fedimint-wallet-client/src/lib.rs

```diff
@@ -916,6 +916,22 @@ impl WalletClientModule {
         }}))
     }
 
+    pub async fn list_peg_in_tweak_idxes(
+        &self,
+    ) -> anyhow::Result<BTreeMap<TweakIdx, PegInTweakIndexData>> {
```

Comment:
why does this return a result ? @dpc

Reply:
https://github.com/fedimint/fedimint/pull/7449